consensus/src/dag/dag_fetcher.rs

```diff
@@ -163,21 +166,34 @@ impl RpcHandler for FetchRequestHandler {
     fn process(&mut self, message: Self::Request) -> anyhow::Result<Self::Response> {
         let dag_reader = self.dag.read();
 
+        // `Certified Node`: In the good case, there should exist at least one honest validator that
```

Comment:
f+1 honest, no?

Reply:
I was thinking f is byzantine, f is under a network partition during fetch and another f did not sign. wrong?

Reply:
Then you mean one honest that has the data in very 2f+1 quorum/